packages/cli/src/ui/components/shared/__snapshots__/RadioButtonSelect.test.tsx.snap

```diff
@@ -0,0 +1,34 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`<RadioButtonSelect /> > renders a list of items and matches snapshot 1`] = `
```

Comment:
one of the important edge cases is >10 items in the list. Add one test that verifies that case.